kube/manifest/do-migration/migration-pv-data-snapshot.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+  name: migration-seed-do-snapshot
+  namespace: starcoin-main
+spec:
+  source:
+    persistentVolumeClaimName: starcoin-volume-starcoin-0
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)